scripts/deployment/travis_before_deploy.sh

```diff
@@ -12,6 +12,33 @@ validate_env_var() {
     done
 }
 
+# ContentaCMS Files To Delete List Function
+# This function receives no argument
+# List of least minimum files needed for Contenta CMS to work
+contentacms_files_to_delete_list() {
+     list=(
+    '.circleci'
+    '.babelrc'
+    'docs'
```

Comment:
I'm pretty sure it won't take much space after ZIP. However I think people will look for the docs online anyways.